### PR TITLE
feat(api): rename schedule models fields

### PR DESCRIPTION
**Pull request by jooola**

### CODE REVIEW

File: api/libretime_api/schedule/models/schedule.py [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
 
 
+class ScheduleManager(models.Manager):
+    def valid(self):
+        """
+        A schedule item is valid if it starts before the end of the show
+        instance it is in.
+        """
+        # return self.starts_at < self.instance.ends_at
+        return self.filter(starts_at__lt=models.F("instance__ends_at"))
```
[the remark this answers is not in the document]
Comment (jooola) [reply api/libretime_api/schedule/models/schedule.py:11]: Maybe we can use  inbound/outbound  or in showframe/out of showframe.

File: api/libretime_api/schedule/models/show.py [hunk 2]
```diff
@@ -3,46 +3,156 @@
...
         blank=True,
         null=True,
+        db_column="autoplaylist_id",
+    )
+    auto_playlist_enabled = models.BooleanField(db_column="has_autoplaylist")
+    auto_playlist_repeat = models.BooleanField(db_column="autoplaylist_repeat")
+
+    hosts = models.ManyToManyField(
+        "core.User",
+        through="ShowHost",
```
[the remark this answers is not in the document]
Comment (paddatrapper) [reply api/libretime_api/schedule/models/show.py:68]: For the moment I think we have to to ensure compatibility with the PHP

File: api/libretime_api/schedule/models/schedule.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jooola) [reply api/libretime_api/schedule/models/schedule.py]: Found 'overbooked' in the legacy code comments, I'll use it.

**Comment by jooola** [comment]
Thanks !

**Pull request merged**